package/MDAnalysis/coordinates/PDB.py

```diff
@@ -987,13 +999,19 @@ def _write_timestep(self, ts, multiframe=False):
 
         .. versionchanged:: 0.7.6
            The *multiframe* keyword was added, which completely determines if
-           MODEL records are written. (Previously, this was decided based on the
-           underlying trajectory and only if ``len(traj) > 1`` would MODEL records
-           have been written.)
+           MODEL records are written. (Previously, this was decided based on
+           the underlying trajectory and only if ``len(traj) > 1`` would
+           MODEL records have been written.)
 
         .. versionchanged:: 1.0.0
-           ChainID now comes from the last character of segid, as stated in the documentation.
-           An indexing issue meant it previously used the first charater (Issue #2224)
+           ChainID now comes from the last character of segid, as stated in
+           the documentation. An indexing issue meant it previously used the
+           first charater (Issue #2224)
+
+        .. versionchanged:: 2.0.0
+           When only record_type attribute is present, instead of using ATOM
+           for both ATOM and HETATM, HETATM record types are properly written
+           out (Issue #1753).
```

Comment:
```suggestion
           When only record_type attribute is present, instead of using ATOM_
           for both ATOM_ and HETATM_, HETATM_ record types are properly written
           out (Issue #1753).
```

Reply:
@RMeli, thanks! In order to add suggested changes:
1. Should I also repeat `.. _ATOM` and `.. _HETATM` in `_write_timestep` to make it available? Or keeping it at the class level is enough (so also no need to repeat `.. _MODEL` and `.. _ENDMDL`, and move `.. _NUMMDL` to the class level).
2. Should I correct older 0.7.6 entry containing `MODEL` without `_`?

Reply:
> Should I also repeat .. _ATOM and .. _HETATM in _write_timestep to make it available? Or keeping it at the class level is enough (so also no need to repeat .. _MODEL and .. _ENDMDL, and move .. _NUMMDL to the class level).

From a local build, I don't think you need to redefine _ATOM/_HETATM, however do build the docs locally and test it out, it's always a recommended thing to do whenever doing docstring changes.

> Should I correct older 0.7.6 entry containing MODEL without _?

Can do, wouldn't hurt.

Reply:
Hi @mieczyslaw

1. As far as I know only one link definition is needed, so keeping it at the class level should be enough. (I think @IAlibay just confirmed this locally.)
2. I didn't want to burden you with suggested changes on documentation you did not write, but it would be great if you could fix that too while you are at it. =)

Reply:
Would it be OK to:
1) move the links to PDB v.3.3?
2) Move to the class level also links in `def HEADER`, `def TITLE`, `def REMARK`, etc. ?
3) `def REMARK` contains `.. _REMARK` and `.. REMARK (update)`, is the latter in use?

Reply:
I will keep 3.2 as it is, added missing _CRYST1 link, move to the class level, sorted out multiple REMARK, REMARKS etc.